test/unit/views/IleQueryViewSpec.scala

```diff
@@ -37,24 +39,47 @@ class IleQueryViewSpec extends ViewSpec {
 
     "render page header" in {
 
-      view.getElementById("title").text() mustBe messages("ileQuery.title")
+      view.getElementsByClass("govuk-label--xl").first().text() mustBe messages("ileQuery.title")
     }
 
     "render error summary" when {
 
       "no errors" in {
 
-        view.getErrorSummary mustBe empty
+        val govukErrorSummary: Element = view.getElementsByClass("govuk-error-summary__title").first()
+
+        Option(govukErrorSummary) mustBe None
       }
 
       "some errors" in {
-        page(IleQueryForm.form.withError("error", "error.required")).getErrorSummary mustBe defined
+
+        val errorView = page(IleQueryForm.form.withError("error", "error.required"))
+
+        val govukErrorSummary = errorView.getElementsByClass("govuk-error-summary__title").first()
+
+        govukErrorSummary.text() mustBe messages("error.summary.title")
       }
     }
 
+    val govukListElement = view.getElementsByClass("govuk-list").first()
+
     "contains submit button" in {
 
-      view.getElementById("submit").text() mustBe messages("site.continue")
+      view.getElementsByClass("govuk-button").first().text() mustBe messages("site.continue")
+    }
+
+//    TODO better way to signal pending tests
+//    "contains link to create empty mucr" in {
+//
+//      govukListElement.getElementsByClass("govuk-link").get(0).text() mustBe messages("ileQuery.link.emptyMucr")
+//    }
+
```

Comment:
Please remove this test case, as it is not going to be implemented any time shortly. There is no point to have commented test cases laying around.